### DISCUSSION TITLE
Using my domain from cPanel on Vercel.

### DISCUSSION THREAD

**Discussion by 4emar** [vercel/community#7223] [Help]
### Summary

Hi everyone, 

I have a question regarding custom domain usage on Vercel. 
My domain is registered to cPanel, and I want to use that domain for my deployment on Vercel. 
I am following the instructions from Vercel for Type A and Type CNAME

Set the following record on your DNS provider to continue:

Type

A
Name

@
Value

76.76.21.21

I changed the values in ZONE EDITOR on cPanel for both domains www.mydomain.com (Type CNAME) and domain.com (Type A), but nothing is happening on Vercel side (still refreshing). Can someone help me with this connectivity, am I doing it wrong or?

### Steps to Reproduce

/

**Comment by amirun573** [comment]
Not sure if work but try using that contain www from vercel. I tried this and it is working instead of using value that contain 76.76.21.21

**Comment by pawlean** [comment]
DNS changes can take some time to propagate globally, typically up to 24-48 hours. It's been 2 weeks since this post - did you manage to get it to work?

[a comment by an automated account is not shown]
